### PR TITLE
feat(infiniteGrid): infiniteGrid handles movement for scroll position

### PR DESCRIPTION
**Pull request by sculove**
### Details
- infinteGrid moves scroll position when prepeding

### Preferred Reviewer
@kishu  @mixed 

### Reference
ref #126

### CODE REVIEW

File: test/manual/infiniteGird_demo.html [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+				imgSrc: "http://naver.github.io/egjs-experiment/infiniteGridService/demo/img/" + ( ( (v + 1) % 60) + 1 ) + ".jpg",
+				href: "http://naver.com/",
+				desc: "Cras justo odio..."
+			};
+		});
+		return {items: items};
+	}
+};
+
+$(document).ready(function() {
```
Comment (netil) [inline test/manual/infiniteGird_demo.html]: just wrap function as
```js
$(function() { ... });
```
which act like ready (http://api.jquery.com/jquery/#jQuery3)

Comment (sculove) [inline test/manual/infiniteGird_demo.html]: @netil ok. I apply it

**Comment by netil** [comment]
LGTM

**Comment by kishu** [comment]
It so nice!
LGTM

**Pull request closed without merging**